table/shell/src/main/java/alluxio/cli/table/command/TransformTableCommand.java

```diff
@@ -72,7 +72,9 @@ public Options getOptions() {
   public String getDescription() {
     return "Transform files representing a structured table under an Alluxio directory."
         + "\n\n"
-        + "Files are coalesced and converted to parquet format.";
+        + "Files are coalesced and converted to parquet format."
+        + "\n\n"
+        + "Definition is in format like 'write(hive).option(hive.num.files, 100)'.";
```

Comment:
Can we point users to some kind of regex about how to specify these options, or what the valid values are? I don't entirely understand the format based on the docs and usage string here.

Reply:
due to the restriction of parser's implementation, currently, only a limited set of definition is supported, I added more explanation.